tests/foreman/sys/test_hot_backup.py

```diff
@@ -46,6 +46,17 @@ def tmp_directory_cleanup(connection, *args):
         connection.run('rm -rf /tmp/{0}'.format(name))
 
 
+def directory_size_compare(connection, full_dir, inc_dir):
```

Comment:
I meant that when you say `compare` it is not obvious that it is True when first greater than second and vice versa